packages/x-components/src/x-modules/tagging/service/__tests__/session.service.spec.ts

```diff
@@ -1,46 +1,38 @@
+import { InMemoryStorageService } from '@empathyco/x-storage-service';
 import { DefaultSessionService } from '../session.service';
-import { MockedStorageService } from './utils/mock-storage.service';
 
 describe('testing session id service', () => {
-  const prefix = 'test';
-  const mockedStorageService = new MockedStorageService(prefix);
+  const mockedStorageService = new InMemoryStorageService();
   const sessionService = new DefaultSessionService(mockedStorageService, 1);
+  const storageKey = sessionService.SESSION_ID_KEY;
+
+  const getItemSpy = jest.spyOn(mockedStorageService, 'getItem');
+  const setItemSpy = jest.spyOn(mockedStorageService, 'setItem');
+  const removeItemSpy = jest.spyOn(mockedStorageService, 'removeItem');
 
   beforeEach(() => {
     jest.clearAllMocks();
   });
 
   it('creates a new session id when calling getSessionId and a session does not exist', () => {
     const session = sessionService.getSessionId();
-    expect(mockedStorageService.getPrefix()).toBe(prefix);
-    expect(mockedStorageService.getSessionIdSpy).toHaveBeenCalledTimes(1);
-    expect(mockedStorageService.setSessionIdSpy).toHaveBeenCalledTimes(1);
-    expect(mockedStorageService.setSessionIdSpy).toHaveBeenCalledWith(
-      'session-id',
-      expect.any(String),
-      expect.any(Number)
-    );
+    expect(getItemSpy).toHaveBeenCalledTimes(1);
+    expect(setItemSpy).toHaveBeenCalledTimes(1);
+    expect(setItemSpy).toHaveBeenCalledWith(storageKey, expect.any(String), expect.any(Number));
     expect(session).toHaveLength(21);
     expect(session).toMatch(/[A-Za-z0-9_-]/);
   });
 
   it('returns an existing session id', () => {
-    mockedStorageService.mockSessionIdValue('abcd-1234');
-    const session = sessionService.getSessionId();
-    expect(mockedStorageService.getPrefix()).toBe(prefix);
-    expect(mockedStorageService.getSessionIdSpy).toHaveBeenCalledTimes(1);
-    expect(mockedStorageService.setSessionIdSpy).toHaveBeenCalledTimes(1);
-    expect(mockedStorageService.setSessionIdSpy).toHaveBeenCalledWith(
-      'session-id',
-      'abcd-1234',
-      expect.any(Number)
-    );
-    expect(session).toBe('abcd-1234');
+    sessionService.getSessionId();
```

Comment:
What's the difference between this and above test? They say different things, but they are testing the same.

Reply:
absolute nothing I think. This is contained above.

Reply:
Sorry for my lack of words. I meant that with your change you made this test equal to the above one. In the origin they were testing different things.

With this simple test I think it should be fine. I tried to extend it to also test the TTL which wasn't tested before, but as the `InMemoryStorageService` ignores it, this would imply more changes.

```ts
  it('returns an existing session id', () => {
    const session1 = sessionService.getSessionId();
    const session2 = sessionService.getSessionId();
    expect(session1).toEqual(session2);
  });
```

Reply:
added that